### PR TITLE
[IZPACK-1425] User is not notified about temp file installer lock existence during automated installation

### PR DESCRIPTION
**Pull request by hiteshbosamiya**

### CODE REVIEW

File: izpack-installer/src/main/java/com/izforge/izpack/installer/requirement/LockFileChecker.java [hunk 1]
```diff
@@ -141,13 +162,10 @@ protected boolean handleLockFile(File file)
             // FIXME Avoid deleting lock for other running instances by using some pool of locks
             // (this is just a workaround to clean up)
             file.deleteOnExit();
+            return true;
```
Comment (reinhapa) [inline izpack-installer/src/main/java/com/izforge/izpack/installer/requirement/LockFileChecker.java:165]: I would leave the single point of return the value

File: izpack-installer/src/main/java/com/izforge/izpack/installer/requirement/LockFileChecker.java [hunk 2]
```diff
@@ -123,10 +125,29 @@ public boolean check()
      */
     protected boolean handleLockFile(File file)
     {
-        boolean result = true;
-        logger.fine("Lock File Exists, asking user for permission to proceed.");
```
Comment (reinhapa) [inline izpack-installer/src/main/java/com/izforge/izpack/installer/requirement/LockFileChecker.java:127]: I would leave the single point of return the value

File: izpack-installer/src/main/java/com/izforge/izpack/installer/requirement/LockFileChecker.java [hunk 3]
```diff
@@ -123,10 +125,29 @@ public boolean check()
...
         StringBuilder msg = new StringBuilder();
         String appName = installData.getInfo().getAppName();
+        if (Installer.getInstallerMode() == Installer.INSTALLER_AUTO) {
+            logger.fine("Lock File Exists.");
+            msg.append("The " + appName + " installer you are attempting to run seems to have a copy already running.");
+            msg.append(" This could be from a previous failed installation attempt or you may have accidentally");
+            msg.append(" launched the installer twice. The recommended action is to wait for the other copy of the");
+            msg.append(" installer to start. If you are sure there is no other copy of the installer running, you can");
+            msg.append(" delete the lock file \"");
```
Comment (reinhapa) [inline izpack-installer/src/main/java/com/izforge/izpack/installer/requirement/LockFileChecker.java:136]: Do we not need any new lines here? Also if we have the same line feeds as within the GUI, extracting the identical parts into a separate method would make sense.

**Review by reinhapa — CHANGES_REQUESTED** [review]

**Comment by hiteshbosamiya** [comment]
As per IZPACK-963, we need to localize the messages. Closing this PR and raise another PR once https://github.com/izpack/izpack/pull/773 is merged

**Pull request closed without merging**